### ISSUE TITLE
lab-07-2 vs lab-07-3

### ISSUE THREAD

**Issue by ipqnojug** [hunkim/DeepLearningZeroToAll#197]
It shows that without minmax lab-07-2 doe not work. The problem in these tow codes is a simple linear regression problem. In fact, it has a good solution by using traditional statistical method without minmax.

My questions are:
1) Thouth algorithm in  lab-07-3 convergences by using minmax in advance, the prediction results do not match the real values of y. So in practice, how can we explain this ? (By the way, it is well known that using traditional statistical method, the prediction results which I get are close to the real values of y, and that it is unnecessary to transoform the data in advance.)
2) If I have a new data (i.e. x1 = [828, 833, 908100, 828]), how can I get the prediction y1?

**Comment by kkweon** [comment]
1. If it's scikit-learn, you just have to call `inverse_transform(y)` method or whatever. However, lab-07-2 wasn't refactored yet. So, you have to modify the function to return necessary info such as min and max. Then you can invert the scale back.
2. You have to min-max scale as you did in training data since min-max is just for scaling. It's purely for optimization purpose nothing else. If it were the normalization, you have to normalize with training data though.
